src/Core/Components/DateTime/CalendarExtended.cs

```diff
@@ -177,4 +180,5 @@ private string ToTitleCase(string value)
     {
         return Culture.TextInfo.ToTitleCase(value);
     }
+
```

Comment:
Could you remove this extra empty line?

Reply:
Sorry, I didn't see it just now. It has been modified to include it. Do I need to create a new PR to organize the commit?

Reply:
No, the last commit was fine